names/data/names_data.hcl

```diff
@@ -0,0 +1,10429 @@
+    provider_name_upper = "IdentityStore"
+    human_friendly      = "SSO Identity Store"
+  }
+
+  endpoint_info {
+    endpoint_api_call        = "ListUsers"
+    endpoint_api_params      = "IdentityStoreId: aws_sdkv2.String(\"d-1234567890\")"
+  }
+
+  resource_prefix {
+    correct = "aws_identitystore_"
+  }
+
+  provider_package_correct = "identitystore"
+  doc_prefix               = ["identitystore_"]
+  brand                    = "AWS"
+}
+
+service "ssooidc" {
+
+  cli_v2_command {
+    aws_cli_v2_command           = "sso-oidc"
+    aws_cli_v2_command_no_dashes = "ssooidc"
+  }
+
+  sdk {
+    id             = "SSO OIDC"
+    client_version = [1]
+  }
+
+  names {
+    aliases             = [""]
+    provider_name_upper = "SSOOIDC"
+    human_friendly      = "SSO OIDC"
+  }
+
+  client {
+    go_v1_client_typename = "SSOOIDC"
+  }
+
+  resource_prefix {
+    correct = "aws_ssooidc_"
+  }
+
+  provider_package_correct = "ssooidc"
+  doc_prefix               = ["ssooidc_"]
+  brand                    = "AWS"
+  not_implemented          = true
+}
+
+service "storagegateway" {
+
+  sdk {
+    id             = "Storage Gateway"
+    client_version = [1]
+  }
+
+  names {
+    aliases             = [""]
+    provider_name_upper = "StorageGateway"
+    human_friendly      = "Storage Gateway"
+  }
+
+  client {
+    go_v1_client_typename = "StorageGateway"
+  }
+
+  endpoint_info {
+    endpoint_api_call        = "ListGateways"
+  }
+
+  resource_prefix {
+    correct = "aws_storagegateway_"
+  }
+
+  provider_package_correct = "storagegateway"
+  doc_prefix               = ["storagegateway_"]
+  brand                    = "AWS"
+}
+
+service "sts" {
+
+  sdk {
+    id             = "STS"
+    client_version = [2]
+  }
+
+  names {
+    aliases             = [""]
+    provider_name_upper = "STS"
+    human_friendly      = "STS (Security Token)"
+  }
+
+  client {
+    skip_client_generate = true
+  }
+
+  env_var {
+    deprecated_env_var = "AWS_STS_ENDPOINT"
+    tf_aws_env_var     = "TF_AWS_STS_ENDPOINT"
+  }
+
+  endpoint_info {
+    endpoint_api_call        = "GetCallerIdentity"
+  }
+
+  resource_prefix {
+    actual  = "aws_caller_identity"
+    correct = "aws_sts_"
+  }
+
+  provider_package_correct = "sts"
+  doc_prefix               = ["caller_identity"]
+  brand                    = "AWS"
+}
+
+service "support" {
+
+  sdk {
+    id             = "Support"
+    client_version = [1]
+  }
+
+  names {
+    aliases             = [""]
+    provider_name_upper = "Support"
+    human_friendly      = "Support"
+  }
+
+  client {
+    go_v1_client_typename = "Support"
+  }
+
+  resource_prefix {
+    correct = "aws_support_"
+  }
+
+  provider_package_correct = "support"
+  doc_prefix               = ["support_"]
+  brand                    = "AWS"
+  not_implemented          = true
+}
+
+service "swf" {
+
+  sdk {
+    id             = "SWF"
+    client_version = [2]
+  }
+
+  names {
+    aliases             = [""]
```

Comment:
Not a blocker, we should also allow omitting `names.aliases` in most cases, since most services don't have aliases